tests/ui/asm/aarch64/type-check-4.rs

```diff
@@ -12,7 +13,7 @@ struct Simd256bit([f64; 4]);
 
 fn main() {}
 
-// Constants must be... constant
+// Constants should be allowed to use statics
```

Comment:
```suggestion
// Constants should be allowed to use the *value* of immutable statics.
```

Reply:
Applied

Reply:
This got superseded by the later change to use a `static mut`. So probably it's best to restore the original comment: `Constants must be... constant`

Reply:
Reverted.